### ISSUE TITLE
Reference to removed 'include/libast/Makefile'

### ISSUE THREAD

**Issue by rakslice** [mej/libast#1]
Files are made reference to in the source that aren't checked in
include/libast/Makefile.am

**Comment by rakslice** [comment]
I didn't notice some of the missing headers at first because I had a previous libast-0.7 build installed on the system

**Comment by rakslice** [comment]
Okay, I think I understand what is happening now. Based on the contents of `include/Makefile.am`, it looks like the intent was for `include/Makefile.am` to take over the duties of `include/libast/Makefile.am`.

But the old file is still referenced in `configure.ac`.
```
$ autoreconf
```
...
```
configure.ac:38: error: required file 'include/libast/Makefile.in' not found
```

Just doing `touch include/libast/Makefile.ac` gets past this error.

**Comment by rakslice** [comment]
Looks like this was fixed in 92df7e62e1b85cf1698e4f6f4e6c4933c115b13a
